package.json

```diff
@@ -6,7 +6,7 @@
   "types": "index.d.ts",
   "main": "index.js",
   "scripts": {
-    "test": "tap ./test"
+    "test": "tap ./test --jobs=1"
```

Comment:
why limit the concurrency?  Are the tests not parallelizable?

Reply:
Whoops, good catch! I was trying to get around using `setTimeout` to wait for the response to be fully processed for the last two tests and put that flag there as a result.